<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersViewForm</class>
 <widget class="QWidget" name="UsersViewForm">
  <property name="windowTitle">
   <string>Users</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_top">
     <item>
      <widget class="QPushButton" name="btnAddUser">
       <property name="text">
        <string>Add User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditUser">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteUser">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_top">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSearchUser">
       <property name="placeholderText">
        <string>Search user...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblUsers">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>